main.c

```diff
@@ -1140,6 +1144,7 @@ int main(int argc, char **argv)
 				NULL  // user date free func
 		);
 
+	tcmu_unblock_netlink();
```

Comment:
Please add this at the 'close_fd:' label to unblock netlink at failures and other abnormal terminations due to signals.

Reply:
Good catch, fixed.